apps/root/src/pages/token-profile/market-stats/index.tsx

```diff
@@ -13,12 +13,16 @@ const StyledMarketStatsContainer = styled(BackgroundPaper).attrs({ variant: 'out
     display: flex;
     flex-direction: column;
     padding: ${spacing(6)};
-    gap: ${spacing(10)};
+    gap: ${spacing(8)};
     height: 100%;
     min-height: ${spacing(75)};
     `}
 `;
 
+const StyledBodyBold = styled(Typography).attrs({ variant: 'bodyBold' })`
+  line-height: 1;
```

Comment:
why does this need a different lineHeight?

Reply:
default for bodyBold is 2, and it adds too much height to the text and therefore the whole container. Now it can match the design